modules/install/rke2_stop_start_instace.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash  
+#set -x
+echo "$@"
+type=$1
```

Comment:
Let's add a comment here on what this variable is for. Could we call it instanceState?